src/app/mail/mail-settings/mail-settings.component.scss

```diff
@@ -561,3 +561,27 @@ $error-red: #e94e4c;
     padding: 14px 1.5rem;
   }
 }
+
+.timezone-filter {
+  width: 100%;
+}
+
+.timezone-block {
```

Comment:
move it to component.styles.scc,
don't use ng deep
```
app-mail-settings {
.timezone-block {
....css here
}
}

Reply:
moved to components, global css.